Cargo.toml

```diff
@@ -50,7 +50,7 @@ split-debuginfo = 'packed'
 
 [features]
 default = ["rustc-demangle", "cpp_demangle", "std-object", "fallible-iterator", "smallvec"]
-std = ["gimli/std"]
+std = ["gimli/std", "cpp_demangle/std"]
```

Comment:
This doesn't appear to be needed, and forces `cpp_demangle` enabled.

Reply:
Hmm, it appears that `addr2line` just tosses errors from `cpp_demangle`, so yeah, that seems true.